src/Input/ToggleSwitch/ToggleSwitch.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+
+import { Icon, SecondaryColor } from '../../index';
+import { Toggle, ToggleBall, ToggleSwitchWrapper, ToggleSwitchProps } from './';
+
+const ToggleSwitch: React.FunctionComponent<ToggleSwitchProps> = ({
+  defaultActive,
+  activeIconName,
+  inactiveIconName,
+  activeBackgroundColor,
+  inactiveBackgroundColor,
+  activeBallColor,
+  inactiveBallColor,
+  onChange,
```

Comment:
Pass any remaining props as `...defaultProps` and pass that to the `Toggle`  component.

```
<Toggle 
  otherProps
  {...defaultProps}
/>
```

This allows the dev to add other props such as `style`.